charts/prometheus/README.md

```diff
@@ -61,6 +61,26 @@ helm upgrade [RELEASE_NAME] [CHART] --install
 
 _See [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) for command documentation._
 
+### To 19.0
+
+Version 19.0.0 uses [prometheus-pushgateway chart version 2.0.0](https://github.com/prometheus-community/helm-charts/tree/main/charts/prometheus-pushgateway#to-200).
+Version 2.0.0 also adapted [Helm label and annotation best practices](https://helm.sh/docs/chart_best_practices/labels/).
+
+Before you update, please delete the `prometheus-pushgateway` StatefulSet or Deployment need to be deleted before upgrade:
+
+```bash
+# If `runAsStatefulSet: false` (this is the default):
+kubectl delete deploy -l app=prometheus-pushgateway
+
+# Or if `runAsStatefulSet: true`:
+# kubectl delete sts -l app=prometheus-pushgateway
+
+# In 18.x
+kubectl scale deploy prometheus-server --replicas=0
```

Comment:
thats likely not need.
have you tested it?

Reply:
Yes. need it.
I have tested the upgrades.

Otherwise, prometheus-18.4.0 pods will continue to linger and prometheus-19.0.0 will crashLoopBackOff

### crashLoopBackOff

```
kind create cluster
helm install prometheus oci://registry/my/prometheus  --version 18.4.0
kubectl delete deploy -l app=prometheus-pushgateway
helm upgrade prometheus oci://registry/my/prometheus --version  19.0.0
```

![image](https://user-images.githubusercontent.com/10738333/205412582-3b4edca4-9567-4751-be34-6481c05b17d9.png)

### Its OK

```
kind create cluster
helm install prometheus oci://registry/my/prometheus  --version 18.4.0
kubectl delete deploy -l app=prometheus-pushgateway
kubectl scale deploy prometheus-server --replicas=0
helm upgrade prometheus oci://registry/my/prometheus --version  19.0.0
```

![image](https://user-images.githubusercontent.com/10738333/205413232-008bb0c6-9121-4a6d-9328-b6d0a6abe4e5.png)